meta-ostro/conf/local.conf.sample

```diff
@@ -35,6 +26,9 @@
 # For Intel Edison:
 #MACHINE ?= "edison"
 #
+# For BeagleBone and BeagleBone Black:
```

Comment:
In meta-ostro/conf/local.conf.sample<https://github.com/ostroproject/meta-ostro/pull/57#discussion_r56383806>:

> @@ -35,6 +26,9 @@
>  # For Intel Edison:
>  #MACHINE ?= "edison"
>  #
> +# For BeagleBone and BeagleBone Black:


We don't support original beaglebone (I doubt even if it's available to purchase anymore), only BBB.

Jussi mentionned both work, do we only test the BBB?